packages/base/package.json

```
@@ -17,6 +17,10 @@
     "directory": "packages/base"
   },
   "exports": {
+    "./dist/ssr-dom.js": {
+      "browser": "./dist/ssr-dom.js",
+      "node": "./dist/ssr-dom-shim.js"
```

```suggestion
      "node": "./dist/ssr-dom-shim.js",
      "default": "./dist/ssr-dom.js"
```
I would recommend to add a `default` fallback here as well, e.g. for testing frameworks like jest

---

makes sense, but shouldn't it be the node file? the browser file is empty and jest will break in that case.

---

I think when using UI5 Web Components, the jest environment will most probably some jsdom environment where all these globals are already present. Hence I would rather vote for making the empty file the default to not accidentally include the shim in browser for the sake of bundle size.